### ISSUE TITLE
Support for mocha retries from mocha config

### ISSUE THREAD

**Issue by rakeshpareek01** [danielstjules/mocha.parallel#53]
module.exports = {
  'allow-uncaught': false,
  'async-only': false,
  bail: false,
  'check-leaks': false,
  color: true,
  delay: false,
  diff: true,

  **retries: 2,**
};

Good to have support for failed test retries

**Comment by steph-lebl** [comment]
I agree with @rakeshpareek01! We are using mocha.parallel for [concurrent api testing](https://stle-code.medium.com/concurrent-api-tests-d84f7a29f0dc?source=friends_link&sk=843339381eaf77195f8522449c907550) and having the possibility to retry will help dealing with flaky tests in the build process.

**Comment by steph-lebl** [comment]
Hi @rakeshpareek01!

Here is the PR for adding retries to mocha.parallel:
https://github.com/danielstjules/mocha.parallel/pull/54

If your interested in the [concurrent api testing](https://stle-code.medium.com/concurrent-api-tests-d84f7a29f0dc?source=friends_link&sk=843339381eaf77195f8522449c907550), the tooling that supports the approach already includes mocha.parallel retries. When a test does not does not succeed on the first time (flaky), the full error in JSON format is available in the test report.
Ex:
https://github.com/VilledeMontreal/mocha-concurrent-api-tests/blob/master/example/README.md#test-report-exemple-when-a-test-fails-or-does-not-succeed-on-the-first-time-flaky

Please give a star to https://github.com/VilledeMontreal/mocha-concurrent-api-tests if it helps you!

Have a nice day!
